### PR TITLE
Merge #1eae173fafa9b29 from CICE-Consortium

### PR DESCRIPTION
**Pull request by apcraig**
## PR checklist
- [X] Short (1 sentence) summary of your PR: 
    Merge #1eae173fafa9b29 from CICE-Consortium
- [X] Developer(s): 
    apcraig
- [X] Suggest PR reviewers from list in the column to the right.
- [X] Please copy the PR test results link or provide a summary of testing completed below.
    This passes all testing on cheyenne with Icepack and CICE.  Regression vs current Consortium main indicates Icepack is bit-for-bit except modal tests (expected).  CICE is bit-for-bit except restart files have different values of Tf on non-ice areas (expected), log files show bit-for-bit in those cases.
- How much do the PR code changes differ from the unmodified code? 
    - [X] bit for bit with Consortium main
    - [ ] different at roundoff level
    - [ ] more substantial 
- Does this PR create or have dependencies on CICE or any other models?
    - [ ] Yes
    - [X] No
- Does this PR add any new test cases?
    - [ ] Yes
    - [X] No
- Is the documentation being updated? ("Documentation" includes information on the wiki or in the .rst files from doc/source/, which are used to create the online technical docs at https://readthedocs.org/projects/cice-consortium-cice/.)
    - [X] Yes
    - [ ] No, does the documentation need to be updated at a later time?
        - [ ] Yes
        - [ ] No 
- [ ] Please provide any additional information or relevant details below:

This merge
- changes answers for fsd runs.
- updates some optional argument implementation (bug not arguments)
- add netcdf ice driver restarts (does not affect column physics)

### CODE REVIEW

File: doc/source/user_guide/ug_implementation.rst [hunk 1]
```diff
@@ -174,13 +174,15 @@ variable ``dumpfreq``. The namelist variable ``ice_ic`` contains the
 pointer to the filename from which the restart data is to be read and 
 the namelist option ``restart`` must be set to ``.true.`` to use the file.
 ``dump_last`` namelist can also be set to true to trigger restarts automatically
-at then end of runs.
+at then end of runs. The default restart file format is binary and is set
```
Comment (eclare108213) [inline doc/source/user_guide/ug_implementation.rst:177]: change 'then' to 'the'.
Also, the 'set by setting' language is a little weird.  How about 

> The default restart file format is binary, set in namelist as ``restart_format = 'bin'``. For netCDF, set ``restart_format = 'nc'``.

?

File: doc/source/user_guide/ug_implementation.rst [hunk 2]
```diff
@@ -197,7 +199,8 @@ written, the USE_NETCDF C preprocessor directive must be set during compilation.
 is done by setting ``ICE_IOTYPE`` to ``netcdf`` in **icepack.settings**.  In addition,
 the machine env and Macros files must include support for compilation with NetCDF.  The
 ``icepack.setup -s`` option ``ionetcdf`` will set the ICE_IOTYPE to netcdf, which turns on 
-the USE_NETCDF C preprocessor.  ``ionetcdf`` also sets the ``history_cdf`` flag to true.
+the USE_NETCDF C preprocessor.  ``ionetcdf`` also sets the ``history_format`` and 
+``restart_format`` flags to 'nc'.
```
Comment (eclare108213) [inline doc/source/user_guide/ug_implementation.rst:203]: It looks like set_nml.ionetcdf is empty now.   Is that true?

**Review by eclare108213 — CHANGES_REQUESTED** [review]

**Comment by eclare108213** [comment]
I realize that this PR is for changes that have already been made in the Consortium code, so it's not the right place to make corrections.

**Comment by apcraig** [comment]
I'll fix the documentation on the Consortium version when I have a chance.  Agree it's best not done here.

File: doc/source/user_guide/ug_implementation.rst (hunk 2 above)
Comment (apcraig) [reply doc/source/user_guide/ug_implementation.rst:203]: Correct, will fix that on Consortium version as well.

**Review by eclare108213 — APPROVED** [review]

**Pull request merged**